spec/integration/kibana/tags.spec.yml

```
@@ -0,0 +1,43 @@
+##
+## Describes the specification for a routing rules yml file
+##
+spec:
+  # Everything under here follows JSON schema (https://json-schema.org/), written as YAML for readability
+  type: array
+  description: Tags automatically added to saved object assets.
+  items:
+     type: object
+     additionalProperties: false
+     properties:
+       text:
+         description: Tag name.
+         type: string
+       asset_types:
+         description: Asset types where this tag is going to be added.
```

So the tag will be added to all the assets of these types included in the package?

Maybe I would rephrase this then to something like:
```suggestion
         description: This tag will be added to all the assets of these types included in the package.
```

---

Yes, that would be more explicit.

@juliaElastic just to confirm, if the same tag is defined in other package. Assets from both packages would be using the same `tag`?

---

According to the original isuse https://github.com/elastic/kibana/issues/152814 it looks like that tag could be the same (reused) in other packages.

> When Fleet installs a package that includes these top-level tags, it should generate the tag + asset references using a unique ID for the tag object. This ID should be a unique identifier based on the downcased text value to avoid collisions and duplications. For example, if two packages define a SecurityExternalIntegrations and securityexternalintegrations tag, these would result in only one tag object being created in Kibana.

---

Updated `asset_types` and `asset_ids` to include this reference

---

@jsoriano @juliaElastic please check if that description makes sense, thanks in advance!